src/main/java/Main.java

```diff
@@ -20,14 +25,18 @@ public static void main(String[] args) {
         Cryptograph cipher = new CommonCryptograph();
```

Comment:
Really clever use of cryptography concepts from previous computer science courses.